docs/installation/aks.md

```diff
@@ -0,0 +1,212 @@
+```bash
+az aks nodepool add \
+  --resource-group "${AZURE_RESOURCE_GROUP}" \
+  --cluster-name "${CLUSTER_NAME}" \
+  --name "${GPU_NODE_POOL_NAME}" \
+  --node-count "${GPU_NODE_COUNT}" \
+  --node-vm-size "${GPU_VM_SIZE}" \
+  --enable-cluster-autoscaler \
+  --min-count 1 \
+  --max-count 3
+```
+
+## 5. Get AKS Credentials
+
+Download and merge the kubeconfig for your AKS cluster:
+
+```bash
+az aks get-credentials \
+  --resource-group "${AZURE_RESOURCE_GROUP}" \
+  --name "${CLUSTER_NAME}" \
+  --overwrite-existing
+```
+
+## 6. Install the NVIDIA Device Plugin (for GPU Clusters)
+
+If you created a GPU node pool, install the NVIDIA device plugin so Pods can request GPUs:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v0.17.1/deployments/static/nvidia-device-plugin.yml
+```
+
+Confirm the DaemonSet is running:
+
+```bash
+kubectl get daemonset nvidia-device-plugin-daemonset -n kube-system
+```
+
+## 7. Install KubeAI
+
+Add the KubeAI Helm repository and update:
+
+```bash
+helm repo add kubeai https://www.kubeai.org
+helm repo update
+```
+
+**(Optional)** If you plan to use private or TOS-protected models from Hugging Face, export your token:
+
+```bash
+export HUGGING_FACE_HUB_TOKEN=<YOUR_HF_TOKEN>
+```
+
+Install KubeAI:
+
+```bash
+helm upgrade --install kubeai kubeai/kubeai \
+  --set secrets.huggingface.token="${HUGGING_FACE_HUB_TOKEN}" \
+  --wait
+```
+
+### Customizing Resource Profiles
+
+If you are running GPU nodes (e.g. `Standard_NC48ads_A100_v4`), you may need to define custom resource profiles or edit existing ones. In your Helm values, you can specify a node selector matching your GPU pool labels:
+
+```yaml
+resourceProfiles:
+  nvidia-azure-a100:
+    nodeSelector:
+      kubernetes.io/hostname: <optional-custom-node-label>
+    limits:
+      nvidia.com/gpu: "1"
+    requests:
+      nvidia.com/gpu: "1"
+```
+
+Then use `resourceProfile: nvidia-azure-a100:1` when installing models. For details, see the [Configure Resource Profiles](../how-to/configure-resource-profiles.md) guide.
+
+## 8. Deploy Models
+
+You can now deploy your models using either:
+
+- The [kubeai/models Helm chart](../how-to/install-models.md#installing-models-with-helm), or
+- Directly applying Model manifests with `kubectl apply -f`.
+
+For instance, to enable a CPU-based text generation model and a GPU-based text embedding model, you might create a `kubeai-models.yaml`:
+
+```yaml
+catalog:
+  llama-3.1-8b-instruct-fp8-l4:
+    enabled: true
+    engine: VLLM
+    resourceProfile: nvidia-azure-a100:1
+    minReplicas: 1
+
+  phi4-gpu:
+    enabled: true
+    features: [TextGeneration]
+    url: hf://microsoft/phi-4
+    engine: VLLM
+    resourceProfile: nvidia-azure-a100:1
+    minReplicas: 1
+```
+
+Then apply:
+
+```bash
+helm install kubeai-models kubeai/models \
+    -f kubeai-models.yaml
+```
+
+Wait for the Pods to appear and become ready:
+
+```bash
+kubectl get pods --watch
+```
+
+## 9. Testing
```

Comment:
we can remove this section since it's covered in this guide: https://www.kubeai.org/how-to/configure-text-generation-models/

Reply:
Sure, this one has been removed